README.md

```diff
@@ -45,7 +45,7 @@ Thanks to [Codable Routing](https://developer.ibm.com/swift/2017/10/30/codable-r
 extension Grade: Model { }
 ```
 
-Now that your `Grade` struct conforms to `Model`, after you [set up](#Create-Your-Database) your database connection pool and create a table sync, you automatically have access to a slew of convenience functions for your object.
+Now that your `Grade` struct conforms to `Model`, after you [set up](#create-your-database) your database connection pool and create a table sync, you automatically have access to a slew of convenience functions for your object.
```

Comment:
Perhaps you could remove the set up link and change the text to something like the following?

“Now that your `Grade` struct conforms to `Model`, after you have set up your database connection pool and created a database table, you automatically have access to a slew of convenience functions for your object.”

Reply:
will do that